src/app/static/src/app/components/plots/bubble/utils.ts

```diff
@@ -31,11 +31,11 @@ export const getFeatureIndicators = function (data: any[],
 
     const result = {} as BubbleIndicatorValuesDict;
     iterateDataValues(data, [sizeIndicator, colorIndicator], selectedAreaIds, filters, selectedFilterValues,
-        (areaId: string, indicatorMeta: ChoroplethIndicatorMetadata, value: number, values: any) => {
+        (areaId: string, indicatorMeta: ChoroplethIndicatorMetadata | undefined, value: number, values: any) => {
```

Comment:
I am not sure undefined should be here. Should there be a need for computation if indicatorMeta is undefined ?